src/safeposix/syscalls/fs_calls.rs

```diff
@@ -2261,109 +2261,203 @@ impl Cage {
         }
     }
 
-    //------------------------------------CHMOD HELPER FUNCTION------------------------------------
-
-    pub fn _chmod_helper(inodenum: usize, mode: u32) {
-        let mut thisinode = FS_METADATA.inodetable.get_mut(&inodenum).unwrap();
-        let mut log = true;
-        if mode & (S_IRWXA | (S_FILETYPEFLAGS as u32)) == mode {
+    /// ### Description
+    ///
+    /// The `_chmod_helper()` is a helper function used by both `chmod_syscall()` and `fchmod_syscall()` 
+    /// to change mode bits that consist of read, write, and execute file permission bits of a file
+    /// specified by an inode obtained from the corresponding caller syscall.
+    ///
+    /// ### Arguments
+    ///
+    /// The `_chmod_helper()` accepts two arguments:
+    /// * `inodenum` - an inode of a file whose mode bits we are willing to change obtained from the caller syscall.
```

Comment:
We need to wrap this and other comments to be <80 chars wide.

Reply:
fixed